readme.md

```diff
@@ -66,7 +66,7 @@ Credits [here](https://kno.wled.ge/about/contributors/)!
 
 Join the Discord server to discuss everything about WLED!
 
-<a href="https://discord.gg/A2FpqFQaRS"><img src="https://discordapp.com/api/guilds/473448917040758787/widget.png?style=banner2" width="25%"></a>
+<a href="https://discord.gg/QAh7wJHrRM"><img src="https://discordapp.com/api/guilds/473448917040758787/widget.png?style=banner2" width="25%"></a>
```

Comment:
And this?

Reply:
I hadn't even seen that this file was uploaded, it came from when I made the clone, I didn't change it... But I'm going to remove it, hehe